how-to/install/enable-network-role.rst

```diff
@@ -0,0 +1,98 @@
+Enable Network Role for Canonical Openstack
+===========================================
+
+
+This how-to guide provides all necessary information to deploy dedicated network nodes in your
+`Canonical OpenStack`_ deployment with Sunbeam.
+
+Overview
+--------
+
+By default, Sunbeam deploys OpenStack with combined control and compute nodes, where networking
+services run alongside compute workloads. The network role allows you to deploy dedicated network
+nodes that handle all networking functions separately.
+
+Network nodes run MicroOVN and the OpenStack network agents (deployed as a subordinate charm
```

Comment:
These are implementation details that are not necessary in a how-to.

We can talk about MicroOVN, but the openstack-network-agents + subordinate charm is too detailed for no net gain for the reader